Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ScreenData.purs

```diff
@@ -132,6 +132,11 @@ initData = {
     , nearByDrivers : Nothing
     , disability : Nothing
     , searchLocationModelData : dummySearchLocationModelData
+    , waitTimeInfo : false
+    , lastSentMessage : { message : "", sentBy : "", timeStamp : "", type : "", delay : 0 }
```

Comment:
create a function for dummyMessage.

Reply:
cannot import the component here, cyclic dependency